e2e/cypress/support/api_commands.js

```diff
@@ -353,10 +353,14 @@ Cypress.Commands.add('apiSaveThemePreference', (value = JSON.stringify(theme.def
 /**
  * Creates a new user via the API, adds them to 3 teams, and sets preference to bypass tutorial.
  * Then logs in as the user
- @param {Object} user - Object of user email, username, and password that you can optionally set. Otherwise use default values
+ @param {Object} user - Object of user email, username, and password that you can optionally set.
+        [Array] teamIDs - list of teams to add the new user to
```

Comment:
These param comments are not valid for jsdoc, should be
```
@param {Array} teamIds - blah blah
@param {boolean} bypassTutorial - blah blah
```

Reply:
Corrected comment format